packages/styles/scss/components/horizontalrule/_horizontalrule.scss

```diff
@@ -0,0 +1,95 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import '~@carbon/themes/scss/themes';
+@import '~carbon-components/scss/globals/scss/vars';
+@import '~carbon-components/scss/globals/scss/helper-mixins';
+@import '~carbon-components/scss/globals/scss/layout';
+@import '~carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/import-once/import-once';
+@import '~carbon-components/scss/globals/scss/css--reset';
+
+@mixin horizontalrule {
+    .#{$prefix}--hr {
+        margin: $spacing-05 0;
+        border: none;
+        height: 1px;
+
+        &__thick {
+            height: 2px;
+        }
+    
+        &__small {
+            width: $layout-03;
+        }
+    
+        &__medium {
+            width: $layout-04;
+        }
+    
+        &__large {
+            width: $layout-05;
+        }
+    
+        &__inset {
+            width: 100%;
+        }
+
+        &__overhung {
+            width: auto;
+
+            @include carbon--breakpoint('sm') {
+                margin-left: -$spacing-05;
+                margin-right: -$spacing-05;
+            }
+
+            @include carbon--breakpoint('md') {
+                margin-left: -$spacing-07;
+                margin-right: -$spacing-07;
+            }
+
+            @include carbon--breakpoint('max') {
```

Comment:
I based this off the IBM Design breakpoint template (https://sketch.cloud/s/ngV7z) where for the max breakpoint we have a `16px` padding + `24px` margin

Reply:
Interesting looks like there is a bug in their grid then because I don't see it in the code. Even more reason I think we should remove layout from this component, and if it's necessary leverage things like `$carbon--grid-gutter`.

https://carbon-elements.netlify.com/grid/examples/preview/

Reply:
Created an issue here on Carbon to track this.

https://github.com/carbon-design-system/carbon/issues/3810

Reply:
great thanks, planning on handling the overhung with grid so no need for the breakpoints anymore